src/blocks/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "namespace": "eightshift-forms",
-  "background": "#900000",
+	"background": "#F6F8F8",
+	"foreground": "#0D3636",
```

Comment:
@goranalkovic-infinum please fix the spacing here (here it should be 2x spaces, not a tab)